spec/models/movie_spec.rb

```diff
@@ -4,4 +4,35 @@
   describe "relationships" do
     it {should belong_to :studio}
   end
+
```

Comment:
Your model tests for these instance methods look good. Don't forget to test the relationships though - we want to see here, for example:
```
it {should have_many :movie_actors}
it {should have_many(:actors).through(:movie_actors)}
```

We should have association tests in the actor spec and movie_actor spec as well